src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -459,7 +525,6 @@ const useGroupInfo = (channelId = null) => {
   const actionLeaveGroup = async () => {
     setOpenModal(false);
     const responseChannelData = await leaveGroup({channelId});
-    console.warn('responseChannelData', responseChannelData);
     try {
       const {channelName} = getChannelListInfo(
         responseChannelData.data,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The console warning statement has been removed which is good for production readiness. The function `actionLeaveGroup` seems to be handling the group leaving action properly.
```diff
- console.warn('responseChannelData', responseChannelData);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->